client/www/index.html

```diff
@@ -69,6 +69,7 @@
 <script src="./scripts/vendor/react/react-with-addons.js"></script>
 
 <!-- d3 -->
+<script src="./scripts/modules/tracing/tracing.viz.module.js"></script>
```

Comment:
This is just a quick hack so the `d3` object doesn't get override. as we discussed you will test a better solution w/ tracing in your env.